Formula/wskdeploy.rb

```diff
@@ -0,0 +1,24 @@
+class Wskdeploy < Formula
+  desc "Apache OpenWhisk project deployment utility"
+  homepage "http://openwhisk.org/"
+  url "https://github.com/apache/incubator-openwhisk-wskdeploy/archive/0.9.7.tar.gz"
+  sha256 "15f586aeb3221e67b583941b988fcc572eebafc64c58d4a2df2787007344f064"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath / "src/github.com/apache/incubator-openwhisk-wskdeploy").install buildpath.children
+    cd "src/github.com/apache/incubator-openwhisk-wskdeploy" do
+      system "godep", "restore"
+      system "go", "build", "-o", bin / "wskdeploy",
+             "-ldflags", "-X main.Version=#{version}"
```

Comment:
Sure

Reply:
Try to make sure we're not bouncing past 80 characters though. In future Homebrew is going to start enforcing line length via robot rather than human and the less warnings on day one the nicer it is for everyone 😄.